py/h2o_wave/server.py

```diff
@@ -73,6 +73,21 @@ def __init__(self, username: str, subject: str, access_token: str, refresh_token
         """The access token of the user."""
         self.refresh_token = refresh_token
         """The refresh token of the user."""
+        self._session_id = session_id
+        """Session identifier. Do not access, internal use only."""
+    
+    async def force_token_refresh(self):
```

Comment:
Plea: can we also return the obtained `access_token`? It would make this method usable as the async token provider used by client libraries.

e.g.:
```python
from typing_extensions import Protocol

class TokenProvider(Protocol):
    async def __call__(self) -> str:
        pass
```

Reply:
I see no problem with that. How shall it be returned? In header or body?

Reply:
I meant just `return access_token` at the end of the method.